Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1/types.go

```diff
@@ -2601,6 +2601,8 @@ type SecurityContextConstraints struct {
 	AllowedCapabilities []Capability `json:"allowedCapabilities" description:"capabilities that are allowed to be added"`
 	// AllowHostDirVolumePlugin determines if the policy allow containers to use the HostDir volume plugin
 	AllowHostDirVolumePlugin bool `json:"allowHostDirVolumePlugin" description:"allow the use of the host dir volume plugin"`
+	// AllowEmptyDirVolumePlugin determines if the policy allow containers to use the EmptyDir volume plugin
+	AllowEmptyDirVolumePlugin *bool `json:"allowEmptyDirVolumePlugin" description:"allow the use of the empty dir volume plugin"`
```

Comment:
This makes me wonder if we should change the upstream proposal to be a black list of plugins rather than a whitelist.